### ISSUE TITLE
[BUG] User settings page cannot have any changes submitted from desktop.

### ISSUE THREAD

**Issue by ICantChooseNames** [MasterSteelblade/WaterfallBugsFeatures#69]
>Despite clicking the submit button under the blacklist tags, I've received an error trying that on desktop whilst trying to switch back to the accessibility font (which I successfully turned on and off in mobile) so I think there's still an issue with saving user settings despite using the button
_Originally posted by @el-o-l in https://github.com/MasterSteelblade/WaterfallBugsFeatures/issues/68#issuecomment-768372226_

Making a new issue page because the User Settings page gives an error if you try to submit any changes, be it through the Submit button or pressing enter in one of the text boxes (Which also would submit changes, if it worked- both methods work on Blog Settings just fine)

**To Reproduce**
1. Go to User Settings
2. Change any setting
3. Try to submit settings

**Expected behavior**
That changes made in user settings be submitted.

**Desktop (please complete the following information):**
- OS: OpenSUSE Leap 15.2
- Browser: Firefox
- Version 78.6.1esr (64-bit)
(for me, anyways ^^'')

**Comment by ICantChooseNames** [comment]
wait I just remembered this post: https://thellere.waterfall.social/post/464694

...is this something that got broken by that?

**Comment by MasterSteelblade** [comment]
Can you tell me what the error message actually is?

**Comment by ICantChooseNames** [comment]
Sorry, should've said what the error was- didn't think to because it was mentioned in the issue this was first brought up in

Error says "CSRF failure. Please refresh the page and try again."

**Comment by MasterSteelblade** [comment]
Possibly fixed - give it a go? The CSRF token was expiring after the first submission, so second submissions should work now.

**Comment by ICantChooseNames** [comment]
Still gives the error :\

**Comment by MasterSteelblade** [comment]
That's odd, it works my end... just to confirm, you refreshed, the relevant pages?

**Comment by ICantChooseNames** [comment]
Yep- regular refresh as well as Ctrl+F5 to be sure

**Comment by MasterSteelblade** [comment]
Fixed!
